src/Libraries/AFILE/Source/amov.c

```diff
@@ -356,9 +356,9 @@ int32_t AmovReadAudio(Afile *paf, void *paudio) {
             // TRACE("%s: got audio chunk in 0x%08x offset", __FUNCTION__, pmi->pcurrChunk->offset);
             mfseek(paf->mf, pmi->pcurrChunk->offset);
             size = mfread(p, MOVIE_DEFAULT_BLOCKLEN, paf->mf);
-            memcpy(paudio + i, p, size);
+            memcpy(((intptr_t) paudio) + i, p, size);
```

Comment:
intptr_t is an integer type, not a pointer type, so passing it to memcpy is probably a bad idea. MSVC won't let you do arithmetic on void pointers, but you should cast it to a uint8_t* rather than intptr_t.